mapflow/functional/layer_utils.py

```diff
@@ -361,6 +361,20 @@ def download_results_file(self, pid) -> None:
             timeout=300
         )
 
+    def download_aoi_file(self, pid) -> None:
+        """
+        Download area of interest and save to a geojson file
+        """ 
+        path = self.temp_dir+str(pid)+'_aoi.geojson'                             
```

Comment:
Don't use string operations on file paths, it is OS-dependent.
Use os.path.join() for strings, or, better stick to pathlib.Path, it is already used somewhere here, and it is better to use it in all the new places. 

You also can refactor to use it for self.temp_dir and all of its usages.

Reply:
It is imported in mapflow.py, sorry for confusion
You can import it to layer_utils as well

For path and string concatenation, use "/" operator, like

path = self.temp_dir/f'{pid}_aoi.geojson'

It will work only if self.temp_dir is a patlib.Path instance, so either go with Path(self.temp_dir) here, or change its type through all the mapflow code

Reply:
If this is confusing, use os.path.join, it's ok